bylaws/3-cf-council.md

```diff
@@ -103,6 +104,8 @@ Efficient participation and decision-making are essential for geographically dis
 - **Voting as a Last Resort:** If a consensus cannot be reached, the CFC resorts to voting. Decisions are then made by a simple majority of the voting CFC members present unless a supermajority is required.
 - **Supermajority Requirement:** Certain critical decisions require a supermajority vote. A supermajority is defined as at least two-thirds of the total number of CFC members.
 
+## Extended Governance Committee (EGC)
+
 ### Matters Requiring Supermajority Vote
 
 The following matters must be approved by a supermajority vote of the CFC:
```

Comment:
I still think that if you use the term "supermajority" you need to specify what that supermajority is (two-third, three-fifth, ...), or who has the power to define what it is and when (on a per-vote basis, periodically by "decree", ...).

It might be useful to mention whether the (super/simple) majority is based on the number of voting members or on the number of members able to vote, though.
Also, I think I saw something about what happens when not enough members are present in the CFC: we'd need to specify whether the same applies to the EGC.

Reply:
I did update to make a few clarifications... it ended up being a bit big, but I think it is an improvement generally. LMK.